tests/meta/test_adaptive_random_forests.py

```diff
@@ -186,59 +133,40 @@ def test_adaptive_random_forests_labels_given():
         X, y = stream.next_sample()
         # Test every n samples
         if (cnt % wait_samples == 0) and (cnt != 0):
-            predictions.append(learner.predict_proba(X)[0])
+            y_proba.append(learner.predict_proba(X)[0])
             true_labels.append(y[0])
-            if np.array_equal(y[0], predictions[-1].argmax()):
-                correct_predictions += 1
 
         learner.partial_fit(X, y)
         cnt += 1
 
-    assert np.alltrue([np.isclose(y_proba.sum(), 1) for y_proba in predictions]), "Probabilities should sum to 1."
-
-    class_probabilities = np.asarray(predictions).squeeze()
-    assert class_probabilities.shape == (49, 2)
-
-    predictions = class_probabilities.argmax(axis=1)
-    last_version_predictions = [1, 1, 0, 1, 1, 0, 0, 1, 0, 1, 1, 1, 1, 0, 1, 0, 1, 1, 0, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1,
-                                1, 1, 1, 1, 1, 0, 1, 1, 1, 0, 1, 0, 0, 0, 1, 1, 1, 0, 0, 0]
-
-    assert np.alltrue(predictions == last_version_predictions)
-
-
-def test_adaptive_random_forests_batch_predict_proba():
-    stream = RandomTreeGenerator(tree_random_state=112, sample_random_state=112, n_classes=2)
-
-    learner = AdaptiveRandomForestClassifier(n_estimators=3,
-                                             random_state=112)
+    assert np.alltrue([np.isclose(y_proba.sum(), 1) for y_proba in y_proba]), \
```

Comment:
The string is a message to provide more information in case of failure: [UsingAssertionsEffectively](https://wiki.python.org/moin/UsingAssertionsEffectively
)

Changed the name in 61e60de9c0db1081c3fe384610d1b14f0bd13a9e

Reply:
Nice! I did not know we could do that with assertions :)